tests/testthat/test-db.R

```diff
@@ -95,21 +95,55 @@ test_that("prevelance must conform", {
   prevalence <- readRDS("data/prevalence.rds")
 
   expect_error(
-    mint_db_check_prevalence(index, prevalence[names(prevalence) != "irsUse"]),
+    mintr_db_check_prevalence(index, prevalence[names(prevalence) != "irsUse"]),
     "Invalid value for 'names(prevalence)':\n  - Missing: irsUse",
     fixed = TRUE)
 
-  expect_silent(mint_db_check_prevalence(index, prevalence))
+  expect_silent(mintr_db_check_prevalence(index, prevalence))
 
   i <- which(prevalence$intervention == "irs")[10]
 
   prevalence$intervention[i] <- "other"
   expect_error(
-    mint_db_check_prevalence(index, prevalence),
+    mintr_db_check_prevalence(index, prevalence),
     "Interventions do not match expected values")
 
   prevalence$intervention[i] <- "No intervention"
   expect_error(
-    mint_db_check_prevalence(index, prevalence),
+    mintr_db_check_prevalence(index, prevalence),
     "Interventions do not match expected values")
 })
+
+
+test_that("building with overwrite replaces all files", {
+  mintr_db_open("data", overwrite = FALSE) # ensure everything here
+  files <- c("data/mintr.db", "data/index.rds", "data/prevalence.rds")
+  old_time <- file.info(files)$mtime
+  old_md5 <- tools::md5sum(files)
+
+  msg <- testthat::capture_messages(mintr_db_open("data", overwrite = TRUE))
```

Comment:
how would you actually force this overwrite to happen? would it be useful to control it via the dockerfile as a build arg?

Reply:
the overwrite is principally useful for developing the import scripts themselves

Reply:
Ok the logic around overwrite has been removed entirely now